projects/core/src/user/facade/user.service.ts

```diff
@@ -453,9 +453,9 @@ export class UserService {
    * Updates the user's email
    * @param uid to be updated
    */
-  updateEmail(uid: string, password: string, newUid: string): void {
+  updateEmail(password: string, newUid: string): void {
```

Comment:
can we rename `newUid` to `newEmail`? We don't want email to be uid, right?

Reply:
`uid` and `email` are one and the same: the there is no email field in the user details, just the uid field where the email is stored.  However, it might be more consistent with the function name to rename it to `newEmail`.

Reply:
After discussion, renaming uid to email has a broader scope than this function and we are leaving it as-is for now.